### DISCUSSION TITLE
Stop / unsubscribe a feed

### DISCUSSION THREAD

**Discussion by nikalexis** [bmoscon/cryptofeed#821] [Q&A]
Is there any command or any workaround that I can stop one feed from the FeedHandler without interrupting the other feeds loaded?

I can see that there is only implementation for add_feed and not e.g. remove_feed...

Thanx you! Great project!

**Comment by bmoscon** [comment]
you'd have to call the stop method on a feed. there is no unsubscribe functionality currently
